src/main/java/com/gtnewhorizons/angelica/compat/nd/Quad.java

```diff
@@ -166,6 +170,12 @@ public void setState(int[] rawBuffer, int offset, BlockRenderer.Flags flags, int
         vectorA.cross(vectorB, vectorC);
 
         normal = ModelQuadFacing.fromVector(vectorC);
+        this.face = ModelQuadFacing.toDirection(normal);
+        try {
+            this.cachedFlags = ModelQuadFlags.getQuadFlags(this);
```

Comment:
Why is this throwing an NPE?

Reply:
If the quad has facing UNASSIGNED, the facing is null and it NPEs. Vintagium just sets it to DOWN in that case (ordinal 0), so I can change it to that